app/components/Home/Home.jsx

```diff
@@ -1,6 +1,6 @@
-import React, { Component } from 'react'
-import { Link } from 'react-router'
-import './home.styl'
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './home.styl';
```

Comment:
We don't use semicolons in our style guide. Install linter in atom plugin 'linter es lint'. And read about .eslintrc file.

Reply:
Done.